cowprotocol/explorer
# feat(batch-graph): special addresses

# Summary

Closes #505 

Tagging special addresses

![image](https://github.com/cowprotocol/explorer/assets/43217/5854f07d-02f3-4d82-adaf-dbfebe6796bd)

For now there are only 2, based on [this thread](https://cowservices.slack.com/archives/C0375NV72SC/p1685627041275649):
- 0xca771eda0c70aa7d053ab1b25004559b918fe662 -> CoW DAO
- 0xA03be496e67Ec29bC62F01a428683D7F9c204930 -> Solver Rewards Safe

Using a different icon for special nodes

# To Test

1. Load a regular batch
* The should be no changes
2. Load a "special" batch like `0x54450bc55a83276344a4e27b951a82d0d5e1927af0126aaac64b1417eb369868`
* The special icon and label should be used
